src/main/java/us/kbase/narrativemethodstore/NarrativeMethodStoreServer.java

```diff
@@ -190,6 +191,10 @@ public static synchronized LocalGitDB getLocalGitDB() throws Exception {
             String dbPwd = nullIfWhitespace(config().get(CFG_PROP_MONGO_PASSWORD));
             System.out.println(NarrativeMethodStoreServer.class.getName() + ": " + CFG_PROP_MONGO_USER +" = " + (dbUser == null ? "<not-set>" : dbUser));
             System.out.println(NarrativeMethodStoreServer.class.getName() + ": " + CFG_PROP_MONGO_PASSWORD +" = " + (dbPwd == null ? "<not-set>" : "[*****]"));
+
+            final boolean retryWrites = config().get(CFG_PROP_MONGO_RETRY_WRITES).equals("ture");
```

Comment:
```suggestion
            final boolean retryWrites = config().get(CFG_PROP_MONGO_RETRY_WRITES).equals("true");
```

Reply:
How did this ever log true?

Reply:
Oops, good catch!

Reply:
Did you check this now logs true as expected?